lms/static/sass/features/_course-experience.scss

```diff
@@ -16,6 +16,7 @@
 
   .message-content {
     @include margin(0, 0, $baseline, $baseline);
+
     position: relative;
```

Comment:
No change needed but curious why you added this line - do you think we should put lines between include/extends statements moving forward?

Reply:
Styleline made me :-)